<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TGVWidget</class>
 <widget class="QWidget" name="TGVWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Parameter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Lambda</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDoubleSpinBox" name="lambda_spinbox">
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Iteration Count</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QSpinBox" name="iteration_count_spinbox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="value">
            <number>10000</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Alpha0</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QDoubleSpinBox" name="alpha0_spinbox">
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Alpha1</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QDoubleSpinBox" name="alpha1_spinbox">
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item alignment="Qt::AlignTop">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Processor</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QRadioButton" name="gpu_radio_button">
           <property name="text">
            <string>GPU</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cpu_radio_button">
           <property name="text">
            <string>CPU</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>GPU Runtime</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QRadioButton" name="cuda_runtime_checkbox">
           <property name="text">
            <string>CUDA</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="thrust_runtime_checkbox">
           <property name="text">
            <string>Thrust</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QPushButton" name="perform_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Perform</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>Algorithm</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QRadioButton" name="tgv2_l1_algorithm_checkbox">
           <property name="text">
            <string>TGV2 L1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tgv2_l2_algorithm_checkbox">
           <property name="text">
            <string>TGV2 L2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tgv1_l1_algorithm_checkbox">
           <property name="text">
            <string>TGV1 L1</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tgv1_l2_algorithm_checkbox">
           <property name="text">
            <string>TGV1 L2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Paint Interval</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="paint_iteration_interval_spinbox">
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
